spec/models/multiple_addresses_spec.rb

```diff
@@ -0,0 +1,25 @@
+# License: AGPL-3.0-or-later WITH Web-Template-Output-Additional-Permission-3.0-or-later
+require 'rails_helper'
+
+RSpec.describe Supporter, type: :model do
+  describe 'multiple address support' do
+    let(:supporter) { create(:supporter)}
+    let(:normal_address) do
+      {
+        address: '120 N Morrison St, Suite 100',
+        city: 'Appleton',
+        state_code: 'WI',
+        country: "United States",
+        zip_code: "54911"
+      }
+    end
+
+    context "where supporter has no address" do 
+      it {
+        
+      }
+    end
```

Comment:
What happens? haha

Reply:
Yikes, it looks like I need to test that more!